src/components/Reviews/ReviewPage/ReviewPage.js

```diff
@@ -53,9 +60,71 @@ const ReviewPage = (props) => {
         }
     }, [router.isReady])
 
-    const submitReview = (body) => {
+    //creates an image for each of the urls the user has submitted
+    const RenderUrls = () => {
+
+        const removeFile = (index) => {
+            fileRecords.splice(index, 1)
+            setFileRecords([...fileRecords])
+        }
+
+        return (
+            fileRecords.map(function (object, i) {
+                return (
+
+                    <div key={object.url} className={styles.photooutercontainer}>
+                        <div className={styles.photocontainer}>
+                            <div className={styles.individualphotoupload}>
+                                <div className={styles.containerdropzone}>
+                                    <img className={styles.image} src={object.url} layout='fill' alt="pic preview" >
+                                    </img>
+                                </div>
+                                <div className={styles.xicon} onClick={() => removeFile(i)}>
+                                    <CancelIcon fontSize="small"></CancelIcon>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+                )
+            })
+        )
+
+    }
+    //submission of the actual review
+    const submitReview = async (body) => {
+
         setIsSubmitDisabled(true);
 
+        async function uploadPhoto(file) {
+
+            const filename = await encodeURIComponent(file.name);
+            const res = await fetch(`/apibackend/s3-upload?file=reviews/${filename}`);
```

Comment:
need to change this to /api. in general we should use ${rootDomain} tho, so that this stuff can just be centralized in one place. will change it since you're on the road